x-pack/plugins/maps/public/url_generator.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  TimeRange,
+  Filter,
+  Query,
+  esFilters,
+  QueryState,
+  RefreshInterval,
+} from '../../../../src/plugins/data/public';
+import { setStateToKbnUrl } from '../../../../src/plugins/kibana_utils/public';
+import { UrlGeneratorsDefinition } from '../../../../src/plugins/share/public';
+import { LayerDescriptor } from '../common/descriptor_types/sources';
+
+const STATE_STORAGE_KEY = '_a';
+const GLOBAL_STATE_STORAGE_KEY = '_g';
+const INITIAL_LAYERS_KEY = 'initialLayers';
```

Comment:
Can you use this constant `INITIAL_LAYERS_KEY` when processing parameter key in the function https://github.com/elastic/kibana/blob/master/x-pack/plugins/maps/public/routing/bootstrap/get_initial_layers.js#L48

Reply:
Ok I will add this to the constants then.